tests/cmsis/CMakeLists.txt

```diff
@@ -5,6 +5,10 @@ if(NOT TEST_FAMILIES)
     set(TEST_FAMILIES F0 F1 F2 F3 F4 F7 G0 G4 H7 L0 L1 L4 L5 U5 WB WL)
 endif()
 
+if("MP1" IN_LIST TEST_FAMILIES)
+    message(WARNING "CMSIS for MP1 devices requires HAL, this test is expected to fail for the MP1 family")
+endif()
```

Comment:
Can you please share what is failing ? Just for my understanding

Reply:
This is because `system_stm32mp1xx.c` (in CMSIS) includes `stm32mp1xx_hal.h` without a `#ifdef USE_HAL_DRIVER` guard.

Reply:
thanks.
I opened https://github.com/STMicroelectronics/STM32CubeMP1/pull/12 to ST as this is a n issue on their side from my point of view. So bad they have no "issue" tab for MP1 series. In the meantime I agree to let the warning